src/AFL-1.2.xml

```diff
@@ -0,0 +1,78 @@
+<SPDX name="Academic Free License v1.2" identifier="AFL-1.2" osi-approved="true">
+  <urls>
+    <url>http://opensource.linux-mirror.org/licenses/afl-1.2.txt</url>
```

Comment:
I don't know that this is the right place to check for this, at least right now. @sam-ellis do you want to check with @jlovejoy ? In the interest of moving things along, I think I'll go ahead and merge this license, and assume that this URL issue will get worked out through another process.